gamecenter/logger.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+from collections import OrderedDict
+import json
+import logging
+
+import arrow
+from flask import request, g
+
+
+class RankTimedRotatingFileHandler(logging.handlers.TimedRotatingFileHandler):
+    """
+    Handles our log with care so the poor child knows where to go
+
+    It will change what is appended to the filename based on the time it's rotated.
+    Suffixes will appear in the form %Y-%m-%d_%H-%M-%S E.g.:
+    /var/log/rank/log-2015-11-04_00-00-00
+
+    Every midnight, the logs rotate to a new file
+    """
+    def __init__(self, filename, backupCount=0, encoding=None, delay=False):
+        kwargs = {
+            'filename': filename,
+            'backupCount': backupCount,
+            'encoding': encoding,
+            'delay': delay,
+            'utc': True,
+            'when': 'midnight',
+            'interval': 1
+            }
```

Comment:
never seen this kind of indentation before but [I guess it's a thing](http://stackoverflow.com/questions/6388187/what-is-the-proper-way-to-format-a-multi-line-dict-in-python)